torch/jit/__init__.py

```diff
@@ -1141,6 +1141,12 @@ def __setattr__(self, attr, value):
                 # parent module. To do this, we create a ConstModuleList, which is itself a module, that
                 # contains each of these modules as submodules. The ConstModuleList then
                 # is set as an attribute of the parent module.
+
+                # allow Sequential Models to nest within ModuleLists
+                for i in range(len(value)):
```

Comment:
Could we refactor this so that this works recursively? (i.e. ModuleList of ModuleList of Sequential)